types/filters/parse.go

```diff
@@ -68,6 +70,26 @@ func ToParam(a Args) (string, error) {
 	return string(buf), nil
 }
 
+func ToParamWithVersion(version string, a Args) (string, error) {
+	// this way we don't URL encode {}, just empty space
+	if a.Len() == 0 {
+		return "", nil
+	}
+
+	// for daemons older than v1.10, filter must be of the form map[string][]string
+	buf := []byte{}
+	err := errors.New("")
+	if version != "" && versionpkg.Version(version).LessThan(versionpkg.Version("1.10")) {
```

Comment:
My bad, I forgot to update this part. Also, is it acceptable to add a version update function in the client?